### PR TITLE
Make nuget ready.

### PR DESCRIPTION
**Pull request by SebastianStehle**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Authors>Liam Riddell,Sebastian Stehle</Authors>
+    <Description>An unofficial port of MJML (by MailJet) to .NET Standard. This project is currently in experimental state and should not be used in a production environment.</Description>
```
Comment (LiamRiddell) [inline Directory.Build.props:4]: We should probably change this? As it's not *as* experimental as the original?

File: Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <Copyright>MIT</Copyright>
+    <PackageTags>email,mjml</PackageTags>
```
Comment (LiamRiddell) [inline Directory.Build.props:6]: Add `templates` tag?

File: Directory.Build.props (hunk 1 above)
Comment (LiamRiddell) [reply Directory.Build.props:4]: We could change this to be slightly more catchy before "release"

File: Directory.Build.props [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/LiamRiddell/MJML.NET</PackageProjectUrl>
```
Comment (LiamRiddell) [inline Directory.Build.props:8]: We'll need to change this to point to the right location.

File: Directory.Build.props [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <RepositoryUrl>https://github.com/LiamRiddell/MJML.NET.git</RepositoryUrl>
```
Comment (LiamRiddell) [inline Directory.Build.props:9]: Same as above.

File: Directory.Build.props (hunk 2 above)
Comment (SebastianStehle) [reply Directory.Build.props:6]: Yes, why not.

File: Directory.Build.props (hunk 3 above)
Comment (SebastianStehle) [reply Directory.Build.props:8]: What is the right location?

Comment (LiamRiddell) [reply Directory.Build.props:8]: Good question. I don't mind :)

Comment (SebastianStehle) [reply Directory.Build.props:8]: Okay, then we can just keep it here. But can you add your nuget key to the secrets?

Comment (LiamRiddell) [reply Directory.Build.props:8]: @SebastianStehle, I can't access repository settings to add the secret.

Comment (SebastianStehle) [reply Directory.Build.props:8]: I tried to add you again, but it seems I cannot give you permissions. Can you send it to sebastian@squidex.io?

Comment (LiamRiddell) [reply Directory.Build.props:8]: Is my user marked as admin?

Comment (SebastianStehle) [reply Directory.Build.props:8]: I don't know how.

**Pull request merged**